### ISSUE TITLE
Arrow support

### ISSUE THREAD

**Issue by storopoli** [farr/MCMCChainsStorage.jl#4]
Could we have Arrow support?

I think it would need to add one more dependency (`Arrow.jl`) and another overload method `save` and `load` using `.arrow`.

**Comment by farr** [comment]
Yes, that would be great!  I don't think I'm going to be able to do it in the near future; but I'd be happy to review a merge request!

**Comment by storopoli** [comment]
Yes, I could try in the summer break. Could you assign this issue to me so I can keep track of it?

**Comment by farr** [comment]
Done.
